plugins/obs-qsv11/common_directx11.h

```diff
@@ -37,4 +27,4 @@ void CleanupHWDevice();
 void SetHWDeviceContext(CComPtr<ID3D11DeviceContext> devCtx);
 CComPtr<ID3D11DeviceContext> GetHWDeviceContext();
 void ClearYUVSurfaceD3D(mfxMemId memId);
-void ClearRGBSurfaceD3D(mfxMemId memId);
\ No newline at end of file
+void ClearRGBSurfaceD3D(mfxMemId memId);
```

Comment:
This change is unrelated.

Reply:
I've removed the addition of the `\n` a the end of this file.